MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,186 +1,98 @@
-    private var statisticService: StatisticService?
+    // MARK: - private functions
     
-    // метод конвертации, который принимает моковый вопрос и возвращает вью модель для экрана вопроса
-    private func convert(model: QuizQuestion) -> QuizStepViewModel {
-        let question = QuizStepViewModel(
-            image: UIImage(data: model.image) ?? UIImage(),
-            question: model.text,
-            questionNumber: "\(currentQuestionIndex + 1)/\(questionsAmount)"
-        )
-        return question
-    }
-    // показ вопроса
-    private func show(quiz step: QuizStepViewModel) {
+    func show(quiz step: QuizStepViewModel) {
+        imageView.layer.borderColor = UIColor.clear.cgColor
         imageView.image = step.image
         textLabel.text = step.question
         counterLabel.text = step.questionNumber
     }
-
-    // приватный метод, который содержит логику перехода в один из сценариев
-    // метод ничего не принимает и ничего не возвращает
-    private func showNextQuestionOrResults() {
-        if currentQuestionIndex == questionsAmount - 1 {
-            guard let statisticService = statisticService else { return }
-            statisticService.store(correct: correctAnswers, total: questionsAmount)
-            
-            let text = """
-                    Ваш результат: \(correctAnswers)/\(questionsAmount)
-                    Количество сыигранных квизов: \(statisticService.gamesCount)
-                    Рекорд: \(statisticService.bestGame.correct)/\(statisticService.bestGame.total)
-                    (\(statisticService.bestGame.date.dateTimeString))
-                    Средняя точность: \(String(format: "%.2f", statisticService.totalAccuracy))%
-                    """
-            let viewModel = AlertModel(
-                        title: "Этот раунд окончен!",
-                        message: text,
-                        buttonText: "Сыграть ещё раз",
-            completion: {[weak self] _ in
-                            guard let self = self else {return}
-                            self.currentQuestionIndex = 0
-                            self.correctAnswers = 0
-                            // заново показываем первый вопрос
-                            questionFactory?.requestNextQuestion()
-                
-            })
-//            alert(quiz: viewModel)
-            alertPresenter?.showAlert(from: self, quiz: viewModel)
-            
-        } else {
-            currentQuestionIndex += 1
-            // идём в состояние "Вопрос показан"
-            showLoadingIndicator()
-            self.questionFactory?.requestNextQuestion()
-        }
+    
+    func show(quiz result: QuizResultsViewModel) {
+        let message = presenter.makeResultsMessage()
+
+        let viewModel = AlertModel(
+                    title: result.title,
+                    message: message,
+                    buttonText: result.buttonText,
+        completion: {[weak self] _ in
```

Comment:
давай договоримся оставлять фигурным скобкам места
```suggestion
        completion: { [weak self] _ in
```